### PR TITLE
Add production deployment scripts and configuration files

### PR DESCRIPTION
**Pull request by dougUCN**
## Changes
- Modifies allowed hosts and CORS for LANL network
- Adds instructions to `CONTRIBUTING.md` for deployment to production
- Adds a copy of the supervisord process manager file used in production to the repository
- Adds a script for deployment to production (will need to be run on the production server manually)
- Removes additional references to Heroku in the documentation and changes to Railway

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -45,17 +45,59 @@ Ariadne implements [subscriptions-transport-ws](https://github.com/apollographql
...
 The production server has Python 3.6.9 and Node js 16.15.1
 
-### 6. Staging
+To update LANE-server on production, perform the following:
+
+```bash
+# Assuming you are in the $HOME/LANE/LANE-server directory
+chmod u+x deploy-prod/deploy.sh
+./deploy-prod/deploy.sh # Doesn't matter which directory from which you call deploy.sh
+```
```
Comment (khyeh) [inline CONTRIBUTING.md:58]: I don't think this comment is necessary. The example assumes user is in `$HOME/LANE/LANE-server` and that alone should suffice.

File: LANE_server/settings.py [hunk 2]
```diff
@@ -31,13 +31,11 @@
...
     '.up.railway.app',
+    '.lanl.gov',
 ]
 
 # For FE and BE hosted locally on different ports
-CORS_ALLOWED_ORIGINS = [
-    "http://localhost:3000",
-    "http://ec2-44-202-29-2.compute-1.amazonaws.com",
-]
+CORS_ALLOWED_ORIGINS = ['*']
```
Comment (khyeh) [inline LANE_server/settings.py:38]: Is this safe?

**Review by khyeh — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (dougUCN) [reply CONTRIBUTING.md:58]: Comment was removed

File: LANE_server/settings.py (hunk 2 above)
Comment (dougUCN) [reply LANE_server/settings.py]: From what I am reading, it is relatively safe minus a few edge cases. Still, it's not exactly clear cut so I will be just reverting to the hard coded list

**Pull request merged**